### DISCUSSION TITLE
Tagging system in Windows Explorer

### DISCUSSION THREAD

**Discussion by Thrau0** [valinet/ExplorerPatcher#3303] [Ideas]
![Bild_2024-05-21_191820559](https://github.com/valinet/ExplorerPatcher/assets/133631503/42e107dd-4210-4ce9-a56a-c1249709cb29)

How about a tagging system in Windows File Explorer? This system would allow users to add and remove tags via the right-click menu. It should also include options to show or hide tags and make them orderable. Additionally, designing the tags to resemble those in Notion would enhance visibility and usability for the user. I would love a feature like this!

**Comment by pyrates999** [comment]
This is beyond what EP is for, sorry.
